src/deploy/osp_deployer/director.py

```diff
@@ -161,6 +161,19 @@ def upload_update_conf_files(self):
             dest_file = self.home_dir + yaml
             self.upload_file(source_file, dest_file)
 
+            unity_lock = "/unity_container_lock.ini"
+            unity_lock_file = self.settings.lock_files_dir + unity_lock
+            if self.settings.enable_unity_backend is True:
+               cmd = "grep unity_cinder__container_version " + unity_lock_file +
```

Comment:
Shouldn t this be grep **_cinder_unity_container_version_**

Reply:
You are right, I have to fix this. Good catch